File: Serene/Serene.Web/Modules/Common/Imports/ServerTypings/Northwind.CustomerCustomerDemoForm.ts
namespace Serene.Northwind {
    export class CustomerCustomerDemoForm extends Serenity.PrefixedContext {
        static formKey = 'Northwind.CustomerCustomerDemo';

    }

    export interface CustomerCustomerDemoForm {
        CustomerID: Serenity.StringEditor;
        CustomerTypeID: Serenity.StringEditor;
    }

    [['CustomerID', Serenity.StringEditor], ['CustomerTypeID', Serenity.StringEditor]].forEach(x => Object.defineProperty(CustomerCustomerDemoForm.prototype, <string>x[0], { get: function () { return this.w(x[0], x[1]); }, enumerable: true, configurable: true }));
}

File: Serene/Serene.Script/Imports/ServerImports/Northwind.CustomerCustomerDemoForm.cs
using jQueryApi;
using Serenity;
using Serenity.ComponentModel;
using System;
using System.Collections;
using System.Collections.Generic;
using System.ComponentModel;
using System.Runtime.CompilerServices;

namespace Serene.Northwind
{
    public partial class CustomerCustomerDemoForm : PrefixedContext
    {
        [InlineConstant] public const string FormKey = "Northwind.CustomerCustomerDemo";

        public CustomerCustomerDemoForm(string idPrefix) : base(idPrefix) {}

        public StringEditor CustomerID { [InlineCode("{this}.w('CustomerID', Serenity.StringEditor)")] get; private set; }
        public StringEditor CustomerTypeID { [InlineCode("{this}.w('CustomerTypeID', Serenity.StringEditor)")] get; private set; }
    }
}

